glue_jupyter/bqplot/common/tools.py

```diff
@@ -218,34 +218,26 @@ def update_selection(self, *args):
             if self.interact.selected_x is not None and self.interact.selected_y is not None:
                 x = self.interact.selected_x
                 y = self.interact.selected_y
-                # similar to https://github.com/glue-viz/glue/blob/b14ccffac6a5
-                # 271c2869ead9a562a2e66232e397/glue/core/roi.py#L1275-L1297
-                # We should now check if the radius in data coordinates is the same
-                # along x and y, as if so then we can return a circle, otherwise we
-                # should return an ellipse.
+                # We enforce strict circle.
                 xc = x.mean()
                 yc = y.mean()
                 rx = abs(x[1] - x[0])/2
                 ry = abs(y[1] - y[0])/2
-                # We use a tolerance of 1e-2 below to match the tolerance set in glue-core
-                # https://github.com/glue-viz/glue/blob/6b968b352bc5ad68b95ad5e3bb25550782a69ee8/glue/viewers/matplotlib/state.py#L198
-                if np.allclose(rx, ry, rtol=1e-2):
-                    roi = CircularROI(xc=xc, yc=yc, radius=rx)
-                else:
-                    roi = EllipticalROI(xc=xc, yc=yc, radius_x=rx, radius_y=ry)
+                r = (rx + ry) * 0.5
```

Comment:
```suggestion
                r = np.sqrt((rx**2 + ry**2) * 0.5)
```
adds a little computational overhead, but I think using arithmetic mean here could create a circle passing somewhat besides the actual pointer position